src/app/tabs/run-tab.js

```diff
@@ -804,6 +804,14 @@ function settings (container, appAPI, appEvents, opts) {
       rpcUrl: (window.location.protocol + '//' + window.location.hostname + ':8546/'),
       vm: 'ielevm'
     })
+
+    addIfNotExists({
+      name: 'KEVM Testnet',
+      context: 'custom-rpc-kevm-testnet',
+      chainId: undefined,
+      rpcUrl: 'https://kevm-testnet.iohkdev.io:8546/',
```

Comment:
Yeah this is the question I have. I don't know how to change it.  
The rpc url for IELE testnet works because you deployed remix to `https://iele-testnet.iohkdev.io/remix/`, so `https://iele-testnet.iohkdev.io/remix/` is a valid rpc url.  
But I don't know how to set the rpc url for KEVM testnet.